docs/user/tutorials/quickstart.md

```diff
@@ -71,8 +71,8 @@ The commands are the same as you have already used:
 
 ```console
 $ hypper install hypper-charts/fleet --create-namespace
-Installing chart "fleet" as "fleet" in namespace "fleet-system"…
-Done! 👏
+🛳  Installing chart "fleet" as "fleet" in namespace "fleet-system"…
+👏 Done!
```

Comment:
I think we should use examples in the quickstart that use shared-dependencies, otherwise the users won't really see the advantage of hypper vs. helm.

Reply:
That's a good point! But I wonder if it would not overcomplicate the quickstart; if we install `our-app` with shared-deps, then we need to explain what happens and why. And the same for the uninstall process and why the unused shared-deps don't disappear yet. The idea of quickstart was to show how to use the CLI in general, not to showcase all the features of Hypper.

Reply:
I don't think so, you do not need to explain, just use the same example but use a chart that has shared an optional deps so the users sees that this is possible just by looking at the output.

Reply:
Done in https://github.com/rancher-sandbox/hypper/pull/158/commits/2763662cf6ce21f7098fd6a16f46835009b4c661!